src/cci/cci_network.c

```diff
@@ -1352,3 +1444,50 @@ connect_srv (unsigned char *ip_addr, int port, char is_retry, SOCKET * ret_sock,
   *ret_sock = sock_fd;
   return CCI_ER_NO_ERROR;
 }
+
```

Comment:
move **`createSSL ()`** to cci_ssl.c, and
remove unused function **`create_ssl ( )`** in cci_ssl.c